packages/crew_profile_base.rb

```diff
@@ -7,11 +7,9 @@ class Crew_profile_base < Package
   version @_ver
   license 'GPL-3+'
   compatibility 'all'
+  source_url "https://github.com/chromebrew/crew-profile-base/archive/refs/tags/#{@_ver}.tar.gz"
```

Comment:
Simplify to `https://github.com/chromebrew/crew-profile-base/archive/#{@_ver}.tar.gz`?